source/Reference.js

```diff
@@ -20,26 +20,42 @@ module.exports.controlTypeCount = Object.keys(module.exports.ControlScheme).leng
 module.exports.TRASHBOT = {
     HEALTH: {
         SNAKE: 1,
-        TURRET: 1,
+        TURRET: 2,
         SNIPER: 1,
-        TANK: 2
+        TANK: 4
     },
     MOVEMENT: {
-        SPEED: 60,
-        RAGE_MULTIPLIER: 1.5,
+        SPEED: 80,
+        RAGE_MULTIPLIER: 1.75,
         AMPLITUDE: 15,
-        PERIOD: 100
+        PERIOD: 150
     }
 }
 module.exports.DIFFICULTY = [
     {
+        LEVEL: 0,
         HEALTH_MULTIPLIER: 1,
         SPAWN_WAVE: {
-            INTERVAL: 5,
-            MAX_HEIGHT: 6,
-            MAX_WIDTH: 7,
-            MAX_WAVES: 2
+            INTERVAL: 1,
+            MAX_HEIGHT: 8,
+            MAX_WIDTH: 8,
+            MAX_WAVES: 6,
+            PATTERNS_PER_WAVE: 1
         },
-
+        JUNK_FREQUENCY_RANGE: { lower: 4, upper: 7},
+        SCORE_LIMIT: 3
+    },
+    {
+        LEVEL: 1,
+        HEALTH_MULTIPLIER: 1.5,
```

Comment:
I don't think this was part of the original scope. Interesting idea though. @ehgoodenough, thoughts?